lib/shopify_app/login_protection.rb

```diff
@@ -26,6 +26,7 @@ def shop_session
 
     def login_again_if_different_shop
       if shop_session && params[:shop] && params[:shop].is_a?(String) && shop_session.url != params[:shop]
+        session[:shopify] = nil
```

Comment:
yeah its also irrelevant now with bugsnag. If we leave it in we should add a comment